torchgeo/datamodules/flair2.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+#
+# FLAIR dataset is released under open license 2.0
+# ..... https://www.etalab.gouv.fr/wp-content/uploads/2018/11/open-licence.pdf
+# ..... https://ignf.github.io/FLAIR/#FLAIR2
```

Comment:
No need for this, it's already included in the docs. The important thing is that this file is MIT licensed like the rest of TorchGeo.